### ISSUE TITLE
Access best_valid

### ISSUE THREAD

**Issue by elenacuoco** [dnouri/nolearn#66]
Hi,
I would like to get value for best_valid loss outside the EarlyStopping class defined in on_epoch_finished=[
        AdjustVariable('update_learning_rate', start=learn, stop=0.001),
        EarlyStopping(patience=20),

    ],
inside a NeuralNet. Is there a method for getting this value?
Thanks a lot for any help
Elena

**Comment by dnouri** [comment]
Access it using `self.on_epoch_finished[0].best_valid`.  This assumes that EarlyStopping is the first element in `on_epoch_finished`.
